api-reference/10 UI Components/dxGantt/1 Configuration/filterRow.md

```diff
@@ -5,7 +5,20 @@ inheritsType: dxGanttFilterRow
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures the filter row settings.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/Gantt/FilterRow/"
+}
+
+Users can enter a value into the filter row to filter Gantt data. Set the [filterRow.visible](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/filterRow/#visible) property to **true** to display the filter row.
+
+![DevExtreme Gantt - Filter Row](/images/Gantt/filter-row.png)
+
+Each cell in the filter row contains a magnifying glass icon. Click a column's magnifying icon to select a filter operation available for the column. Use the [selectedFilterOperation](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#selectedFilterOperation) property to specify the default filter operation for a column's filter row. 
```

Comment:
"a magnifying icon" - this phrase looks strange... "an icon that magnifies?"
I believe we can simplify this a bit because "a magnifier icon" is the same as "a magnifying glass icon":

```suggestion
Each cell in the filter row contains a magnifier icon. Click a column's magnifier icon to select a filter operation available for the column. Use the [selectedFilterOperation](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#selectedFilterOperation) property to specify the default filter operation for a column's filter row. 
```